functions.php

```diff
@@ -54,18 +60,83 @@ function createBottlesHtml(array $allBottles): string {
     }
     $bottlesHtml = "";
     foreach($allBottles as $bottle) {
-        $bottlesHtml .= '<div class="bottleCard">';
-        $bottlesHtml .= '<h3>' . $bottle['itemname'] . '</h3>';
-        foreach($bottle as $detailName => $detailValue){
-            if($detailName == 'itemname' || $detailName == 'id') {
-                continue;
-            }
-            $bottlesHtml .= '<h4>' . $detailName . '</h4>' . '<p>' . $detailValue . '</p>';
+        $bottlesHtml .=
+            '<div class="bottleCard">
+            <a class="anchor" id="dbId-'.$bottle['id'].'"></a>';
```

Comment:
this anchor isn't linking to anything, you might as well just put that id on something else